service.c

```diff
@@ -0,0 +1,16 @@
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+
+// get the delay time given lambda (average time between processing)
+double get_service_delay(double lambda){
```

Comment:
Sounds good. Please create a header file (`service.h`) so we can include this function in other files.